### PR TITLE
Feature/tulcdm 177 cleanly handle missing digital collection entry

### PR DESCRIPTION
**Pull request by skng5**
Skipping rake task download and convert specs. Need to figure out how to pre-populate the DigitalCollection table before running the specs.

### CODE REVIEW

File: lib/cdm_utils.rb [hunk 1]
```diff
@@ -3,10 +3,23 @@
 require "fileutils"
 
 module CDMUtils
+
+  def self.available_collections
+    digital_collections = DigitalCollection.all.map{ |c| c.collection_alias }
```
Comment (bibliotechy) [inline lib/cdm_utils.rb]: Seems like a great opportunity to use [pluck](http://apidock.com/rails/ActiveRecord/Calculations/pluck) here:
`DigitalCollection.pluck(:collection_alias)`

File: lib/cdm_utils.rb [hunk 2]
```diff
@@ -3,10 +3,23 @@
...
+  end
+
   def self.list(server)
```
Comment (bibliotechy) [inline lib/cdm_utils.rb]: I'm not sure we need to call out to ContentDM at all here. We only want to see items that have active records, right? So doesn't it make sense to just ask active record directly, and skip the contentDM middle man? 
We coudl replace all of that with:
`DigitalCollection.pluck(:collection_alias, :name).to_h`
Am I missing some crucial aspect here?

**Comment by skng5** [comment]
Good catch. Done.

**Comment by bibliotechy** [comment]
I refactored the list method some more because it was still calling out to contentDM. 

I was unable to run tests with the simplecov-csv lines in the Gemfile commented out. Do you plan to remove those permanently?

**Comment by skng5** [comment]
Yes. Unless you think otherwise, I'll do it in this PR so we can move on.

**Pull request merged**